iset.mm

```diff
@@ -7109,6 +7109,19 @@ Many theorems of this section actually hold for stable propositions (see
       ( ( ( ph -> ps ) -> ps ) -> ( ( ps -> ph ) -> ph ) ) ) $=
     ( wi wdc imim1 peircedc syl9r ) ABCZBCBACHACADAHBAEABFG $.
 
+  $( Double negation of excluded middle.  Intuitionistic logic refutes the
+     negation of excluded middle (but does not prove excluded middle) for any
+     formula.  Can also be proved quickly from ~ bj-nnor as in ~ bj-nndcALT .
+     (Contributed by BJ, 9-Oct-2019.) $)
+  nnexmid $p |- -. -. ( ph \/ -. ph ) $=
+    ( wn wo wa pm3.24 ioran mtbir ) AABZCBHHBDHEAHFG $.
+
+  $( Double negation of decidability of a formula.  Intuitionistic logic
+     refutes undecidability (but does not prove decidability) of any formula.
+     (Contributed by BJ, 9-Oct-2019.) $)
+  nndc $p |- -. -. DECID ph $=
+    ( wdc wn wo nnexmid df-dc notbii mtbir ) ABZCAACDZCAEIJAFGH $.
+
```

Comment:
I think these belong the "Decidable propositions" section, not the "Theorems of decidable propositions" section (the latter is typically for theorems having decidability as an antecedent, and a classical tautology as a consequent).

Reply:
I was wondering about that too but I wasn't sure.

Moved.